charts/miggo-operator/templates/delayed-resources/configmap.yaml

```diff
@@ -10,11 +19,11 @@ data:
       name: miggo-instrumentation
     spec:
       exporter:
-        endpoint: http://{{ .Release.Name }}-collector.{{ .Values.collector.collectorNameSpace | default .Release.Namespace }}:{{ .Values.collector.collectorPort }}
+        endpoint: {{ include "collectorAddress" . }}:{{ .Values.collector.collectorPort }}
```

Comment:
What if we wanna transmit directly to Miggo backend collector? 
For example riskified has their own collector so we might wanna transmit to its endpoint or straight to miggo backend

Reply:
currently we don't allow an easy configuration of the collector address, we probably should allow it.
(in any case, this change didn't break this option, its not exists in main branch as well)

Reply:
TBC in another PR